L/libpostal/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libpostal"
+version = v"1.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/openvenues/libpostal.git", "8f2066b1d30f4290adf59cacc429980f139b8545")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/libpostal
+flags=""
+
+if [[ "${proc_family}" != "intel" ]]; then
+    # SSE2 doesn't apply on ARM or PowerPC
+    flags="${flags} --disable-sse2"
+fi
+
+# BinaryBuilder notes that the w64 binaries use the AVX2 instruction set. I haven't
+# been able to fix this, as you can't use -march with BinaryBuilder, and -mno-avx2
+# does not solve the problem.
+
+./bootstrap.sh
+
+# data download handled in Julia
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-data-download ${flags}
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+# aarch64-linux has an assembler error: "Error: conditional branch out of range"
+# This is a rare enough platform we just exclude for now
+platforms = expand_cxxstring_abis(supported_platforms(; exclude=p -> p.tags["os"] == "linux" && p.tags["arch"] == "aarch64"))
```

Comment:
I don't agree aarch64-linux is that rare.  Also, this seems to be https://gcc.gnu.org/legacy-ml/gcc-patches/2018-01/msg01808.html, which was fixed around GCC v8
```suggestion
platforms = expand_cxxstring_abis(supported_platforms())
```